src/styles/index.scss

```diff
@@ -1,3 +1,86 @@
+$stopwatch-size: 80vmin;
+$center-size: 5vmin;
+$minutes-hand-hight: 20vmin + $center-size / 2;
+$minutes-hand-width: 3vmin;
+$seconds-hand-hight: 32vmin + $center-size / 2;
+$seconds-hand-width: 1.5vmin;
+
 body {
   margin: 0;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  @function center-stopwatch-elements($size) {
+    @return ($stopwatch-size * 0.5) - ($size * 0.5);
+  }
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($minutes-hand-width);
+
+    height: $minutes-hand-hight;
+    width: $minutes-hand-width;
+
+    background-color: #0700ff;
+
+    transform-origin: bottom;
+    animation: hand-move 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($seconds-hand-width);
```

Comment:
Similar to line 32, ensure that the `center-stopwatch-elements` function is correctly scoped and accessible for use here.